__init__.py

```
@@ -0,0 +1 @@
+from mysite import *
```

Ruff (0.8.2)

`from mysite import *` used; unable to detect undefined names

(F403)